### PR TITLE
2772 readable message of EOerror

### LINKED ISSUE #2772: Unreadable exception  Ex EOorg.EOeolang.EOstringν62
The following stack trace does not help to find out the mistake in eo code:
```
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.154 s <<< FAILURE! -- in EOorg.EOeolang.EOsqrt_trashTest
[ERROR] EOorg.EOeolang.EOsqrt_trashTest.works -- Time elapsed: 0.087 s <<< ERROR!
EOorg.EOeolang.EOerror$ExError: EOorg.EOeolang.EOstringν62
        at org.eolang.PhSafe.attr(PhSafe.java:92)
        at org.eolang.PhImmovable.attr(PhImmovable.java:81)
        at org.eolang.PhOnce.attr(PhOnce.java:92)
        at org.eolang.PhOnce.attr(PhOnce.java:92)
        at org.eolang.PhLocated.attr(PhLocated.java:118)
        at org.eolang.PhNamed.attr(PhNamed.java:88)
        at org.eolang.PhSafe.attr(PhSafe.java:90)
        at org.eolang.PhImmovable.attr(PhImmovable.java:81)
        at org.eolang.PhOnce.attr(PhOnce.java:92)
        at org.eolang.PhLocated.attr(PhLocated.java:118)
        at org.eolang.PhMethod.lambda$new$0(PhMethod.java:43)
        at org.eolang.Data$Once.lambda$take$0(Data.java:120)

```

### PR DESCRIPTION
**Pull request by levBagryansky**
Closes #2772

---

### Detailed summary
- Added import statements for `StandardCharsets`, `Bytes`, `BytesOf`, and `Dataized` in `EOerror.java`.
- Modified the `ExError` constructor to use the `safeMessage` method to set the error message.
- Added a new private method `safeMessage` in `ExError` to retrieve the message from the enclosure safely.
- Added import statements for `AtComposite`, `AtOnce`, `Data`, `Dataized`, `PhCopy`, `PhDefault`, `PhWith`, `Phi`, `MatcherAssert`, `Matchers`, `Assertions`, `Test`, `ParameterizedTest`, and `MethodSource` in `EOerrorTest.java`.
- Added a new parameterized test method `getsReadableError` in `EOerrorTest.java` to test the `safeMessage` method.
- Added a new parameterized test method `getsReadableByteArrayError` in `EOerrorTest.java` to test the `safeMessage` method with byte arrays as input.
- Added a new inner class `MyError` in `EOerrorTest.java` to create a test case for the `safeMessage` method.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by levBagryansky** [comment]
@maxonfjvipon please review

File: eo-runtime/src/test/java/EOorg/EOeolang/EOerrorTest.java [hunk 1]
```diff
@@ -55,4 +65,67 @@ void makesToxicObject() {
...
+            Matchers.containsString(cnst.toString())
+        );
+    }
+
+    /**
+     * Input arguments for getsReadableError unit test.
+     * @return Stream of arguments.
+     */
+    private static Stream<Object> getTestSources() {
+        return Stream.of(
```
Comment (maxonfjvipon) [inline eo-runtime/src/test/java/EOorg/EOeolang/EOerrorTest.java:89]: @levBagryansky what about byte array?

**Review by maxonfjvipon — APPROVED** [review]
@levBagryansky LGTM! Thanks

**Comment by levBagryansky** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**